Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,22 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Variables
```

Comment:
Don't declare all the variables in one place- delcare them as close as possible where they are used

Reply:
Will do that, thanks for the correction @Almantask !